AppCenterData/AppCenterData/MSData.m

```diff
@@ -661,6 +664,7 @@ - (void)deleteFromCosmosDbWithPartition:(NSString *)partition
   [self performCosmosDbOperationWithPartition:partition
                                    documentId:documentId
                                    httpMethod:kMSHttpMethodDelete
+                                   httpClient:self.httpClientWithRetrier
```

Comment:
This one isn't processing a pending delete, right? Shouldn't this one be without a retryer?

Reply:
this is actually coming from `processPendingOperations` call. for regular delete it uses:
```- (void)deleteDocumentWithID:(NSString *)documentID
                   partition:(NSString *)partition
                writeOptions:(MSWriteOptions *_Nullable)writeOptions
           completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler {```